<!--REQUEST_ID=30020b73-e8ff-4834-bf9b-c76bf3a1631b-->
Failure on x86_64-darwin [(full log)](https://logs.nix.ci/?key=nixos/nixpkgs.48927&attempt_id=35b78a9d-caba-407d-8a55-9b8c5165134a)

Attempted: signify

<details><summary>Partial log (click to expand)</summary><p>

```
sha2.c:952:1: error: only weak aliases are supported on darwin
MAKE_CLONE(SHA512_256Pad, SHA512Pad);
^
./compat.h:30:20: note: expanded from macro 'MAKE_CLONE'
                                __attribute__((alias (#src)))
                                               ^
16 errors generated.
make: *** [<builtin>: sha2.o] Error 1
builder for '/nix/store/6bbfj7c6h65x6l3wnrp90wwc7caqxk17-signify-24.drv' failed with exit code 2
error: build of '/nix/store/6bbfj7c6h65x6l3wnrp90wwc7caqxk17-signify-24.drv' failed
```
</p></details>

--- a/pkgs/tools/security/signify/default.nix
+++ b/pkgs/tools/security/signify/default.nix
@@ -1,0 +1,34 @@
+{ stdenv, fetchFromGitHub, libbsd, pkgconfig }:
+
+stdenv.mkDerivation rec {
+  name = "signify-${version}";
+  version = "24";
+
+  src = fetchFromGitHub {
+    owner = "aperezdc";
+    repo = "signify";
+    rev = "v${version}";
+    sha256 = "0grdlrpxcflzmzzc30r8rvsmamvbsgqyni59flzzk4w5hpjh464w";
+  };
+
+  doCheck = true;
+
+  nativeBuildInputs = [ pkgconfig ];
+  buildInputs = [ libbsd ];
+
+  preInstall = ''
+    export PREFIX=$out
+  '';
+
+  meta = with stdenv.lib; {
+    description = "OpenBSD signing tool";
+    longDescription = ''
+      OpenBSDs signing tool, which uses the Ed25519 public key signature system
+      for fast signing and verification of messages using small public keys.
+    '';
+    homepage = https://www.tedunangst.com/flak/post/signify;
+    license = licenses.isc;
+    maintainers = [ maintainers.rlupton20 ];
+    platforms = platforms.linux;
+  };
+}

--- a/pkgs/top-level/all-packages.nix
+++ b/pkgs/top-level/all-packages.nix
@@ -5277,6 +5277,8 @@ with pkgs;
 
   signal-desktop = callPackage ../applications/networking/instant-messengers/signal-desktop { };
 
+  signify = callPackage ../tools/security/signify { };
+
   # aka., pgp-tools
   signing-party = callPackage ../tools/security/signing-party { };
 

